test/e2e/testing-manifests/storage-csi/mock/csi-mock-driver.yaml

```diff
@@ -93,3 +103,12 @@ spec:
             path: /var/lib/kubelet/plugins_registry
             type: Directory
           name: registration-dir
+        - hostPath:
+            # 'path' is where PV data is persisted on host.
+            path: /var/lib/csi-hostpath-data/
```

Comment:
These tests are going to run in parallel, potentially on the same hosts. We can't use `/var/lib/csi-hostpath-data` on the host.

I don't think any of the mock tests rely on persisting the driver state across pod restarts. Therefore let's not use any hostpath volume. Instead pass `-statedir=/tmp/csi-hostpath-data` as parameter.

https://github.com/kubernetes-csi/csi-driver-host-path/blob/d0d44abf58958cbf2105a56da4968db2cee26291/cmd/hostpathplugin/main.go#L49

Reply:
I changed to use `--statedir` for saving the state.